src/components/dialoghelper/dialogHelper.js

```diff
@@ -0,0 +1,486 @@
+define(['appRouter', 'focusManager', 'browser', 'layoutManager', 'inputManager', 'dom', 'css!./dialoghelper.css', 'scrollStyles'], function (appRouter, focusManager, browser, layoutManager, inputManager, dom) {
+    'use strict';
+
+    var globalOnOpenCallback;
+
+    function enableAnimation() {
```

Comment:
looks like a duplicate file from a rename

Reply:
Could you check if it's still duplicated file? I removed the duplicated files, and it seems that the diff here disapeared, only appearing the renaming.